components/PopulationCalculation.tsx

```diff
@@ -59,8 +63,13 @@ export default function PopulationCalculation() {
     });
 
     if (measureBundle.content) {
-      const measureReports = await Calculator.calculateMeasureReports(measureBundle.content, patientBundles, options);
-      return measureReports.results as fhir4.MeasureReport[];
+      const calculationResults = await Calculator.calculate(measureBundle.content, patientBundles, options);
+      const { results, coverageHTML } = calculationResults;
+      if (coverageHTML) {
+        setClauseCoverageHTML(coverageHTML);
+      }
+      const measureReports = MeasureReportBuilder.buildMeasureReports(measureBundle.content, results, options);
```

Comment:
`calculateMeasureReports` does not return the clause coverage HTML in its output, but the base `calculate` function does. So, instead of using `calculateMeasureReports`, I used the base `calculate` function in order to retrieve the results and the clause coverage HTML. The `results` from `calculate()` are not in the form of a measure report, so I had to apply `MeasureReportBuilder.buildMeasureReports` to the calculation results